travis_ci_operator.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
```

Comment:
@MuhammadIsmailShahzad So either we keep this or remove and get it with curl

Reply:
@zelima 
Will keep this
Was using it with curl before
Added it in the repo in the final PR forgot about removing curl
Will remove the curl command

Reply:
But why removing curl?

Reply:
What if smth changes there?